src/EmployerAccounts/SFA.DAS.EmployerAccounts.Api/AppStart/AddConfigurationOptionsExtension.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using SFA.DAS.Api.Common.Configuration;
+using SFA.DAS.SharedOuterApi.Configuration;
+
+namespace SFA.DAS.EmployerAccounts.Api.AppStart
+{
+    public static class AddConfigurationOptionsExtension
+    {
+        public static void AddConfigurationOptions(this IServiceCollection services, IConfiguration configuration)
+        {
+            services.AddOptions();
+            services.Configure<AccountsConfiguration>(configuration.GetSection(nameof(AccountsConfiguration)));
```

Comment:
This should be:
services.Configure<AccountsConfiguration>(configuration.GetSection("AccountsApiConfiguration"));

as the configuration file is called "AccountsApiConfiguration" where as the class name is AccountsConfiguration

Reply:
Actually I was wrong yours was consistent between the code and config originally! - I will make it more consistent with other outerapi's in my branch